### PR TITLE
Added support for `-o --out` #330

### PR DESCRIPTION
**Pull request by wolfendale**

### CODE REVIEW

File: library/src/main/scala/giter8/TemplateRenderer.scala [hunk 1]
```diff
@@ -21,14 +21,14 @@ import java.io.File
 
 trait TemplateRenderer {
   def render(baseDirectory: File,
-             outputDirectory: File,
+             outputDirectory: Either[File, File],
```
Comment (eed3si9n) [inline library/src/main/scala/giter8/TemplateRenderer.scala]: What does the `Either[File, File]` mean? Does `TemplateRenderer` have to know if it's Left or Right?

File: build.sbt [hunk 2]
```diff
@@ -41,7 +41,6 @@ lazy val root = (project in file("."))
 lazy val app = (project in file("app"))
   .disablePlugins(BintrayPlugin)
   .enablePlugins(ConscriptPlugin, BuildInfoPlugin, SonatypePublish)
-//  .enablePlugins(BuildInfoPlugin, SonatypePublish)
```
Comment (xuwei-k) [inline build.sbt]: Why disable these plugins?

Comment (wolfendale) [reply build.sbt]: The build breaks in Intellij whenever I load the project with the Conscript plugin. Would raise an issue but I'm not sure whether it's an issue in conscript or intellij. Haven't had much of a chance to look into it.

Comment (wolfendale) [reply build.sbt]: Note, this is just removing a commented out line that I'd added during development. It's not actually a change to the build

**Pull request merged**